plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/META-INF/MANIFEST.MF

```diff
@@ -7,13 +7,15 @@ Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.newproject.test
 Bundle-Vendor: Google Inc.
 Bundle-Version: 0.1.0.qualifier
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.newproject
-Require-Bundle: org.junit;bundle-version="4.12.0",
+Require-Bundle: com.google.cloud.tools.eclipse.appengine.flex,
```

Comment:
Import-Package is preferred these days for OSGI reasons. Could we use that instead?